projects/opendr_ws/src/perception/README.md

```diff
@@ -1,46 +1,64 @@
-```
+- ### Increase performance by disabling output
+    Optionally, nodes can be modified via command line arguments, which are presented for each node separately below. Generally, arguments give the option to change the input and output topics, the device the node runs on (CPU or GPU), etc. When a node publishes on several topics, where applicable, a user can opt to disable one or more of the outputs by providing `None` in the corresponding output topic. This disables publishing on that topic, forgoing some operations in the node, which might increase its performance. 
+
+    _An example would be to disable the output annotated image topic in a node when visualization is not needed and only use the detection message in another node, thus eliminating the OpenCV operations._ 
+<!-- - ### Other notes -->
+
+----
+## RGB input nodes
+
+----
+
+### Pose Estimation ROS Node
+
+You can find the pose estimation ROS node python script [here](./scripts/pose_estimation.py) to inspect the code and modify it as you wish to fit your needs. The node makes use of the toolkit's [pose estimation tool](../../../../src/opendr/perception/pose_estimation/lightweight_open_pose/lightweight_open_pose_learner.py) whose documentation can be found [here](../../../../docs/reference/lightweight-open-pose.md).
+
+Instructions for basic usage and visualization of results:
+
+1. Start the node responsible for publishing images. If you have a USB camera, then you can use the `usb_cam_node` as explained in the [prerequisites above](#prerequisites).
+
+2. You are then ready to start the pose detection node:
+    ```shell
+    rosrun perception pose_estimation.py
+    ```
+    The following optional arguments are available:
+   - `-h, --help`: show a help message and exit
+   - `-i or --input_rgb_image_topic INPUT_RGB_IMAGE_TOPIC`: topic name for input RGB image (default=`/usb_cam/image_raw`)
+   - `-o or --output_rgb_image_topic OUTPUT_RGB_IMAGE_TOPIC`: topic name for output annotated RGB image, `None` to stop the node from publishing on this topic (default=`/opendr/image_pose_annotated`)
+   - `-d or --detections_topic DETECTIONS_TOPIC`: topic name for detection messages, `None` to stop the node from publishing on this topic (default=`/opendr/poses`)
+   - `--device DEVICE`: Device to use, either `cpu` or `cuda`, falls back to `cpu` if GPU or CUDA is not found (default=`cuda`)
+   - `--accelerate`: Acceleration flag that causes pose estimation to run faster but with less accuracy
 
-3. You can examine the annotated image stream using `rqt_image_view` (select the topic `/opendr/image_pose_annotated`) or
-   `rostopic echo /opendr/poses`. 
+3. In a new terminal you can view the annotated image stream by running `rosrun rqt_image_view rqt_image_view` and selecting the topic `/opendr/image_pose_annotated` or by running `rostopic echo /opendr/poses`, where the node publishes the detected poses in [OpenDR's 2D pose message format](../ros_bridge/msg/OpenDRPose2D.msg).
```

Comment:
I re-checked the link for the `OpenDRPose2D.msg` file both locally and [on github](https://github.com/opendr-eu/opendr/tree/ros-nodes-docs/projects/opendr_ws/src/perception#pose-estimation-ros-node) on the `ros-nodes-docs` branch and it seems to be working for me. Also verified the relative link and it seems correct. This branch is based on develop where the msg file exists, am i missing something? :smile:

Reply:
You are right. It seems correct.
I only checked on the GitHub interface and it doesn't work for me because it strangely redirects me to the master branch.